src/modules/Dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -1,29 +1,40 @@
 import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
 
+import { fetchImagesList } from "modules/Dashboard/actions/dashboard";
 import Regular from "common/containers/Regular";
 
 import { Button } from "react-bootstrap";
 import { faWrench } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import CardImage from "../CardImage/CardImage";
+import NotesModal from "../NotesModal/NotesModal";
+
+class Dashboard extends Component {
+    componentDidMount() {
+        const { dispatch } = this.props;
+        dispatch(fetchImagesList());
```

Comment:
Use `mapDispatchToProps`, look at [this](https://react-redux.js.org/using-react-redux/connect-mapdispatch).